lib/roda/plugins/dry_view.rb

```diff
@@ -24,6 +24,15 @@ def view(name, options = {})
           options = {context: scope.view_context}.merge(options)
           on to: scope.view_key(name), call_with: [options]
```

Comment:
Instead of adding the `#block_result_body` below (kudos for finding a way to make that work, btw! I had no idea that method existed), I wonder if we can just change the way to invoke the view here, e.g.

```ruby
resolve(scope.view_key(name)) do |view|
  view.({context: scope.view_context}.merge(options)).to_s
end
```

Reply:
that works really well!